src/datepicker/datepicker-navigation.ts

```diff
@@ -65,4 +74,19 @@ export class NgbDatepickerNavigation {
     (event.currentTarget as HTMLElement).focus();
     this.navigate.emit(this.navigation.NEXT);
   }
+
+  getScreenReaderFrom() {
+    const f = this.months[0].firstDate;
+    return this.i18n.getMonthFullName(f.month, f.year) + ' ' + this.i18n.getYearNumerals(f.year);
+  }
+
+  getScreenReaderTo() {
+    const t = this.months[this.months.length - 1].lastDate;
+    return this.i18n.getMonthFullName(t.month, t.year) + ' ' + this.i18n.getYearNumerals(t.year);
+  }
```

Comment:
Aren't these things supposed to be localised (l10n) ? Meaning that depending on the lang, it might be `month + " " + year` or `year + ", " + month` ...

Reply:
Moved implementation to NgbDatepickerI18n. Did it the same way as day aria label.